test/TestDNSSEC.js

```diff
@@ -160,21 +184,21 @@ contract('DNSSEC', function(accounts) {
         type: 'DNSKEY',
         class: 'IN',
         ttl: 3600,
-        data: { flags: 0x0101, algorithm: 253, key: Buffer.from('0000', 'HEX') }
```

Comment:
Why the change in flags?

Reply:
They aren't actually changed but the put in canonical order to match [`rawSignatureData()`](https://github.com/ensdomains/dnsprovejs/blob/master/lib/dns/result.js#L53).